src/Illuminate/Encryption/EncryptionManager.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Illuminate\Encryption;
+
+use Illuminate\Foundation\Application;
+use Illuminate\Support\Str;
+use RuntimeException;
+
+class EncryptionManager
+{
+    /**
+     * @var Application
+     */
+    protected $app;
+
+    /**
+     * @var Encrypter[]
+     */
+    protected $encrypters = [];
+
+    /**
+     * EncryptionManager constructor.
+     * @param Application $app
+     */
+    public function __construct(Application $app)
```

Comment:
Please don't insist on the laravel app instance here, otherwise this code will be broken on Lumen. Please take a look at how the cache manager is implemented and mimic that here.